src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +123,455 @@ const useStyles = makeStyles()(() => ({
+        initialValues={{
+          statusValue: statusValue,
+          pledgeCurrency: amountCurrency,
+          pledgeAmount: amount,
+          pledgeFrequency: frequencyValue,
+        }}
+        validationSchema={appealFormSchema}
+        onSubmit={async (values) => {
+          await onSubmit(values);
+        }}
+      >
+        {({
+          values: {
+            statusValue,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          },
+          handleSubmit,
+          setFieldValue,
+          errors,
+        }): ReactElement => {
+          const modalContact = {
+            id: id,
+            status: statusValue,
+            name: name,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          };
+
+          return (
+            <Form onSubmit={handleSubmit}>
+              <Grid container className={classes.formWrapper}>
+                <Card className={classes.formInner}>
+                  <Grid container>
+                    <Grid item lg={6} md={4} xs={12}>
+                      <Box
+                        display="flex"
+                        p={2}
+                        alignItems="center"
+                        className={classes.left}
+                      >
+                        <Avatar
+                          src=""
+                          style={{
+                            width: theme.spacing(7),
+                            height: theme.spacing(7),
+                          }}
+                        />
+                        <Box display="flex" flexDirection="column" ml={2}>
+                          <Link
+                            data-testid="contactSelect"
+                            underline="hover"
+                            onClick={() => setContactFocus(id, 'Donations')}
+                          >
+                            <Typography variant="h6">{name}</Typography>
+                          </Link>
+                          <Typography>
+                            {`Current: ${statusTitle} ${
+                              amount &&
+                              amountCurrency &&
+                              currencyFormat(amount, amountCurrency, locale)
+                            } ${frequencyTitle}`}
+                          </Typography>
+                        </Box>
+                      </Box>
+                    </Grid>
+                    <Grid item xs={12} md={8} lg={6} className={classes.right}>
+                      <Grid
+                        container
+                        style={{ paddingRight: theme.spacing(1) }}
+                      >
+                        <Grid item xs={12} md={6} lg={12}>
+                          <Box className={classes.boxTop}>
+                            <InputLabel id="status-label">
+                              {t('Status')}
+                            </InputLabel>
+                            <Select
```

Comment:
```suggestion
                            <FormControl><InputLabel id="status-label">
                              {t('Status')}
                            </InputLabel>
                            <Select
```

Reply:
If you wrap your <Select> in <FormControl> it will make the label look nicer.

Reply:
@wjames111 I think this should do it: `<FormControl fullwidth>
<InputLabel id="status-label">
                              {t('Status')}
                            </InputLabel>
                            <Select
                              className={classes.select}
                              placeholder="Status"
                              labelId="status-label"
                              label={t('Status')}
                              inputProps={{
                                'data-testid': 'pledgeStatus-input',
                              }}
                              data-testid="statusSelect"
                              style={{ width: '100%' }}
                              value={statusValue}
                              onChange={(event) =>
                                setFieldValue('statusValue', event.target.value)
                              }
                            >
                              <MenuItem value="" disabled>
                                Status
                              </MenuItem>
                              {statuses.map((status) => (
                                <MenuItem
                                  value={status.value}
                                  key={status.value}
                                  data-testid="statusSelectOptions"
                                >
                                  {status.name}
                                </MenuItem>
                              ))}
                            </Select></FormControl>`